package.json

```diff
@@ -73,7 +73,7 @@
   },
   "devDependencies": {
     "@rollup/plugin-commonjs": "^22.0.2",
```

Comment:
I notice we're doing something special with CameraFlightPath
https://github.com/PropellerAero/cesium/blob/main/package.json#L41

Reply:
WE should Propeller hack this